packages/markdown/component/readme.md

```diff
@@ -3,15 +3,14 @@
 This package brings legacy support for the `<Markdown />` component to all Astro projects.
 
 > The `<Markdown />` component does not work in SSR. Consider [importing Markdown content](https://docs.astro.build/en/guides/markdown-content/#importing-markdown) instead.
```

Comment:
While we’re here we can probably make this a [warning](https://github.com/orgs/community/discussions/16925) instead of a plain blockquote too:

```suggestion
> **Warning**
> The `<Markdown />` component does not work in SSR. Consider [importing Markdown content](https://docs.astro.build/en/guides/markdown-content/#importing-markdown) instead.
```